tracer/test/snapshots/AwsEventBridgeTests.NetCore.SchemaV1.verified.txt

```diff
@@ -0,0 +1,90 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: async-methods,
+    Resource: async-methods,
+    Service: Samples.AWS.EventBridge,
+    Tags: {
+      env: integration_tests,
+      language: dotnet,
+      runtime-id: Guid_1
+    },
+    Metrics: {
+      process_id: 0,
+      _dd.top_level: 1.0,
+      _dd.tracer_kr: 1.0,
+      _sampling_priority_v1: 1.0
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_3,
+    Name: http.client.request,
+    Resource: POST localhost:00000/,
+    Service: Samples.AWS.EventBridge,
+    Type: http,
+    ParentId: Id_2,
+    Tags: {
+      component: HttpMessageHandler,
+      env: integration_tests,
+      http-client-handler-type: System.Net.Http.HttpClientHandler,
+      http.method: POST,
+      http.status_code: 200,
+      http.url: http://localhost:00000/,
+      language: dotnet,
+      out.host: aws_eventbridge,
+      peer.service: aws_eventbridge,
+      span.kind: client,
+      _dd.peer.service.source: out.host
+    }
+  },
+  {
+    TraceId: Id_1,
+    SpanId: Id_4,
+    Name: aws.eventbridge.send,
+    Resource: EventBridge.PutEvents,
+    Service: Samples.AWS.EventBridge,
+    Type: http,
+    ParentId: Id_2,
+    Tags: {
+      aws.agent: dotnet-aws-sdk,
+      aws.operation: PutEvents,
+      aws.requestId: Guid_2,
+      aws.service: EventBridge,
+      aws_service: EventBridge,
```

Comment:
Note: they were on the other snapshots too

Reply:
Okay nevermind: https://github.com/DataDog/dd-trace-dotnet/blob/d09331a2d718bd11b3a51fce19d1eaba1884f909/tracer/src/Datadog.Trace/Tags.cs#L393-L399

It seems that they are on all the AWS snapshots, so probably expected.

Reply:
This is OK due to have because of [this RFC](https://docs.google.com/document/d/1450pjyhzqHTTtEvPyZGzH1cKpca7gOwtxgel_LUsR4w/edit#heading=h.w9tpabmagxvr)

Reply:
Awesome thanks @duncanista!